model/trace/database.yaml

```diff
@@ -238,16 +240,24 @@ groups:
             sensitive information.
         brief: The request body for a [search-type query](https://www.elastic.co/guide/en/elasticsearch/reference/current/search.html), as a json string.
         examples: [ '"{\"query\":{\"term\":{\"user.id\":\"kimchy\"}}}"' ]
+      - ref: db.collection.name
+        requirement_level: recommended
+        brief: The index or data stream against which the query is executed.
+        note: >
+          The query may target multiple indices or data streams, in which case it SHOULD be a comma separated list of those.
+          If the query doesn't target a specific index, this field MUST NOT be set.
+        examples: [ 'my_index', 'index1, index2' ]
       - ref: server.address
       - ref: server.port
-      - ref: db.elasticsearch.cluster.name
-        requirement_level:
-          recommended: >
-            When communicating with an Elastic Cloud deployment, this should be collected from the "X-Found-Handling-Cluster" HTTP response header.
+      - ref: db.namespace
+        note: >
+          When communicating with an Elastic Cloud deployment, this should be collected from the "X-Found-Handling-Cluster" HTTP response header.
```

Comment:
this could prevent us from making `db.namespace` a sampling relevant attribute: https://github.com/open-telemetry/semantic-conventions/issues/1019#issuecomment-2142324021

Reply:
I forgot, we changed MUST to SHOULD here:

> The following attributes can be important for making sampling decisions and SHOULD be provided at span creation time (if provided at all)

so this is not a big problem